packages/main/src/plugin/extension-loader.ts

```diff
@@ -1074,6 +1076,99 @@ export class ExtensionLoader {
       },
     };
 
+    const navigateTo = (navigateRequest: NavigationRequest) => {
+      this.apiSender.send('navigate', navigateRequest);
+    };
+    const navigation: typeof containerDesktopAPI.navigation = {
+      navigateToContainers: async function (): Promise<void> {
+        navigateTo({ page: NavigationPage.CONTAINERS });
+      },
```

Comment:
we should not use function keywords there but use arrow functions
```suggestion
      navigateToContainers: async : Promise<void> => {
        navigateTo({ page: NavigationPage.CONTAINERS });
      },
```